File: wlxx-gateway/src/main/java/com/xzg/wlxx/gateway/GatewayApplication.java
package com.xzg.wlxx.gateway;

import org.mybatis.spring.annotation.MapperScan;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
import org.springframework.boot.autoconfigure.security.servlet.SecurityAutoConfiguration;
import org.springframework.cloud.client.discovery.EnableDiscoveryClient;

/**
 * @author xzgan
 * @datetime 2022/5/25 20:16
 * @package com.xzg.wlxx.gateway
 */

@SpringBootApplication(exclude = {DataSourceAutoConfiguration.class, SecurityAutoConfiguration.class})
@EnableDiscoveryClient
public class GatewayApplication {
    public static void main(String[] args) {
        SpringApplication.run(GatewayApplication.class,args);
    }
}

File: wlxx-gateway/src/main/kotlin/com/xzg/wlxx/gateway/GatewayApplication.kt
package com.xzg.wlxx.gateway

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration
import org.springframework.cloud.client.discovery.EnableDiscoveryClient

@SpringBootApplication(exclude = [DataSourceAutoConfiguration::class])
@EnableDiscoveryClient
object GatewayApplication {
    @JvmStatic
    fun main(args: Array<String>) {
        SpringApplication.run(GatewayApplication::class.java, *args)
    }
}
